packages/jest-config/src/utils.ts

```diff
@@ -248,3 +248,7 @@ export const getSequencer = (
     prefix: 'jest-sequencer-',
     rootDir,
   });
+
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+export const interopRequireDefault = (obj: any): {default: any} =>
```

Comment:
nit: given that this is only used in one place, I don't mind having it just in `readConfigFileAndSetRootDir` to remove an import.

You could even actually inline it directly into the `loadTSConfigFile` function as I think its not being used elsewhere in `jest-config` on purpose (i.e if you stuck this on the require for the js config, it'd break it completely)

Reply:
Seeing the `utils.ts` file I thought that that's the right place, but I guess you are right, I can move it inside `loadTSConfigFile` and just move it later if it's the case.